### DISCUSSION TITLE
News Cache [2023-03-13]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#1770] [Announcements]
### 🚢  Ships

- [Improvements and Fixes](https://vercel.com/changelog/march-2023)

### 💡 Inspiration

- [Why Turborepo is migrating from Go to Rust](https://vercel.com/blog/turborepo-migration-go-rust)
- [Framework-defined infrastructure](https://vercel.com/blog/framework-defined-infrastructure)

**Comment by raakafeh** [comment]
Hello
